src/common/Screen.js

```diff
@@ -122,7 +122,7 @@ class Screen extends PureComponent<Props> {
           contentContainerStyle={[padding && styles.padding]}
         >
           <ScrollView
-            contentContainerStyle={[styles.flexed, centerContent && componentStyles.content, style]}
+            contentContainerStyle={[centerContent && componentStyles.content, style]}
```

Comment:
Now that I can see this more clearly, I realize that it may not be a good idea. This would change the style for *all* `Screen`-using components, not just `AccountPickScreen`, and that may have unwanted effects on other screens – especially since this style was itself introduced as a bugfix in commit 9fa95a9b790ddbac5193f3cce42d9bdd5ef48c3e.

That bugfix seems like something of a hack itself, so removing it might actually be the right thing to do – but that would have to be explained and/or verified (preferably both).

Reply:
Thank you @ray-kraesig ,
>We shouldn't hide the scrollbar, here or on other scrollable pages, as it removes important functionality:

That is a great point, i have created an alternative solution for that,
added some margin for "Add new account" button, so that it escapes from scroll bar overlay,

> This would change the style for all Screen-using components, not just AccountPickScreen, and that may have unwanted effects on other screens – especially since this style was itself introduced as a bugfix in commit 9fa95a9.

>That bugfix seems like something of a hack itself, so removing it might actually be the right thing to do – but that would have to be explained and/or verified (preferably both).

I have not modified Screen this time, as it may give side effects,
If you think  that code needs to be replaced then I will add a separate PR
![ezgif-2-105859cc225a](https://user-images.githubusercontent.com/23723464/75259455-13036f00-580e-11ea-8df6-f0563bce7c0e.gif)